flytekit/configuration/statsd.py

```diff
@@ -4,3 +4,4 @@
 
 HOST = _common_config.FlyteStringConfigurationEntry("statsd", "host", default="localhost")
 PORT = _common_config.FlyteIntegerConfigurationEntry("statsd", "port", default=8125)
+DISABLED = _common_config.FlyteBoolConfigurationEntry("statsd", "disabled", default=False)
```

Comment:
Can you explain why this is necessary?  Were you seeing an error in sagemaker?  If so, what sets it?  Should we set it by default for certain types of jobs?

Reply:
SageMaker currently doesn't support running side-car containers nor custom AMIs.. so statsD is not an option. It's a flag the plugin will set (not flytekit) so it can control what to set it to (if we deploy custom AMIs that has statsD relay on localhost, the plugin can override the config) I don't think flytekit should make assumptions about the execution environment... generally speaking